src/GameWorldSimulator/NeoServer.Game.Creatures/Models/Bases/CombatActor.cs

```diff
@@ -480,6 +497,8 @@ protected void InvokeAttackCanceled()
     public event AddCondition OnAddedCondition;
     public event RemoveCondition OnRemovedCondition;
     public event Attacked OnAttacked;
+    public event HealthChange OnHealthChange;
```

Comment:
can we rename to "OnHealthChanged" and "OnManaChanged" ?

Reply:
yes, fixed on last commit: https://github.com/OpenCoreMMO/OpenCoreMMO/pull/650/commits/860579bf572f65383870d8e11bc2596ff0d53822